server/build/webpack.js

```diff
@@ -39,6 +44,10 @@ export default async function createCompiler (dir, { hotReload = false, dev = fa
       log: false,
       // required not to cache removed files
       useHashIndex: false
+    }),
+    new webpack.optimize.CommonsChunkPlugin({
```

Comment:
Here's minChunks is the place for tuning up whether to move modules to main.js or not. We may need to expose this to the user lately.